CMakeLists.txt

```diff
@@ -220,6 +220,9 @@ ELSE()
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
         ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     )
+if(UNIX)
```

Comment:
The indentation looks wrong here, it should be indented to show it is part of the previous else clasue.